### DISCUSSION TITLE
Execute GoPro Labs QR commands using the API

### DISCUSSION THREAD

**Discussion by aroooy** [gopro/OpenGoPro#419] [Ideas]
Is it possible to execute GoPro QR commands (Camera Extensions, etc.) using the API?

I think it would be convenient to be able to change settings without loading a QR into the GoPro, and I think BLE PROTOBUF is suitable for achieving this.

**Comment by marcotidei** [comment]
I need this!

**Comment by alexivaner** [comment]
Hi guys, want to follow up the progress of this one

**Comment by grahampcurran** [comment]
Any progress on this?
